sphinx/application/data/concepts/repositories/concept-repository-dashboard/src/main/java/chat/sphinx/concept_repository_dashboard/RepositoryDashboard.kt

```diff
@@ -48,6 +48,12 @@ interface RepositoryDashboard {
         challenge: String
     ): Response<Boolean, ResponseError>
 
+    suspend fun saveProfile(
+        relayUrl: String,
```

Comment:
relayUrl should be removed since it's not used.

Reply:
should be resolved in a74452a